src/components/InvoiceCard/InvoiceCardsList.tsx

```diff
@@ -1,62 +1,73 @@
-import React, { useEffect, useState } from 'react'
+import React, { useEffect, useState } from "react";
 import axios from "axios";
-import InvoiceCard from './InvoiceCard';
-import { InvoiceToShow } from '../common';
+import InvoiceCard from "./InvoiceCard";
+import { InvoiceToShow } from "../common";
+import { getCardList } from "components/InvoiceCard/actions";
+import Button from "components/Button/Button";
+import { filterSVG, plusSVG } from "images";
+import FilterBy from "components/FilterBy/FilterBy";
+
+const createNewInvoice = () => {};
 
 const InvoiceCardsList = () => {
+  let [cardListing, setcardListing] = useState([] as InvoiceToShow[]);
+  let [statefilterWindow, setStatefilterWindow] = useState(false);

```

Comment:
Sugestão: usa "isFilterShown" e "setIsFilterShown".

É sempre mais facil utiliziar "is" e "has" para booleans.